config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -266,42 +305,45 @@ processors:
             
       - metric_name: system.swap.paging_ops
         action: update
-        new_name: agent.googleapis.com/swap/io
+        new_name: swap/io
         operations:
           # delete singular type dimension, retaining only direction
           - action: aggregate_labels
             label_set: [ direction ]
             aggregation_type: sum
 
 exporters:
-  stackdriver:  
+  stackdriver/agent:  
+    project:  # stackdriver will use the default GCE credentials
+    metric_prefix: agent.googleapis.com/
+
+  stackdriver/custom:  
     project:  # stackdriver will use the default GCE credentials
```

Comment:
Maybe explicitly put the `metric_prefix` here so its really obvious to people what's happening